core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/HBaseTxClient.java

```diff
@@ -452,7 +438,15 @@ public short abortTransaction(final long transactionID) throws Exception {
           Object commitDDLLock = new Object();
           synchronized(commitDDLLock)
           {
-            commitDDLLock.wait();
+             boolean loopBack = false;
+             do {
+                try {
+                    commitDDLLock.wait();
```

Comment:
Should there be a loopBack = false after the wait call in the case where an exception occurred but then the operation succeeded?